WooCommerce/Classes/ViewRelated/Orders/Order Details/Customer Section/Billing Information/BillingInformationViewController.swift

```diff
@@ -161,6 +161,31 @@ private extension BillingInformationViewController {
     func displayMessageComposerIfPossible(from: UIViewController) {
         messageComposer.displayMessageComposerIfPossible(order: order, from: from)
     }
+
+    /// Create an action sheet that offers the option to copy the email address
+    ///
+    func displayEmailCopyAlert(from: UIViewController) {
+        guard let sourceView = from.view, let email = order.billingAddress?.email else {
+            return
+        }
+
+        let actionSheet = UIAlertController(title: nil, message: nil, preferredStyle: .actionSheet)
+        actionSheet.view.tintColor = .primary
+
+        actionSheet.addCancelActionWithTitle(ContactAction.dismiss)
+        actionSheet.addDefaultActionWithTitle(ContactAction.copyEmail) { [weak self] _ in
+            ServiceLocator.analytics.track(.orderDetailCustomerEmailTapped)
+            self?.sendToPasteboard(self?.order.billingAddress?.email)
+        }
+
+        let popoverController = actionSheet.popoverPresentationController
+        popoverController?.sourceView = sourceView
+        popoverController?.sourceRect = sourceView.bounds
+
+        from.present(actionSheet, animated: true)
+
+        ServiceLocator.analytics.track(.orderDetailCustomerPhoneMenuTapped)
```

Comment:
probably the email menu tapped event?

Reply:
🤦‍♀ yes, the email menu tapped event. Thank you!